spec/comment_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let!(:user) { User.create!(name: 'Test User', posts_counter: 0) }
+  let!(:post) { Post.create!(title: 'Test Post', author: user, comments_counter: 0, likes_counter: 0) }
+
+  it 'increments the comments counter when created' do
+    expect { Comment.create(text: 'Test Comment', user:, post:) }.to change {
+                                                                       post.reload.comments_counter
+                                                                     }.by(1)
+  end
+
+  it 'decrements the comments counter when destroyed' do
+    comment = Comment.create(text: 'Test Comment', user:, post:)
+    expect { comment.destroy }.to change { post.reload.comments_counter }.by(-1)
+  end
```

Comment:
- [ ] Any custom method in your models should have corresponding tests. For example, the `update_comments_counter` method in your Post model should be tested

Reply:
![image](https://github.com/lerfast/blog-app/assets/111823606/8a1b996b-a091-49c7-a1b4-e628200993c1)
Done!!!